samples/routing/data-distribution/sample.md

```diff
@@ -8,6 +8,8 @@ tags:
 - Data distribution
 ---
 
+DANGER: Asynchronous messaging is not an optimal solution for data distribution scenarios. It is usually better to use a dedicated data distribution technology for that purpose, such as a distributed cache or distributed configuration service. This sample aims to serve only as an example of the flexibility in the NServiceBus routing engine.
```

Comment:
@DavidBoike - this disclaimer looks good to me and captures what we wanted in the Lisbon meeting.